content/en/docs/reference/setup-tools/kubeadm/kubeadm-config.md

```diff
@@ -16,7 +16,7 @@ You can execute `kubeadm config view` to view the ConfigMap. If you initialized
 kubeadm v1.7.x or lower, you must use `kubeadm config upload` to create the ConfigMap before you
 may use `kubeadm upgrade`.
 
-In Kubernetes v1.11.0, some new commands were added. You can use `kubeadm config print-default`
+In Kubernetes v1.11.0, some new commands were added. You can use `kubeadm config print`
```

Comment:
This amendment makes it look like the revised command `kubeadm config print` was introduced in Kubernetes v1.11
Is that right?

Reply:
given 1.11 is outside of support we can safely remove this sentence:

> In Kubernetes v1.11.0, some new commands were added.